kitchen-tests/kitchen.vagrant.yml

```diff
@@ -0,0 +1,24 @@
+driver:
```

Comment:
This kitchen config isn't going to actually test the current content of chef/chef. It'll just pull the latest stable build. You probably want to remove it from this PR.

Reply:
Done.